spec/game_spec.rb

```diff
@@ -0,0 +1,187 @@
+require './lib/game'
+require './lib/player'
+
+describe Game do
+  inplay = Game.new
```

Comment:
- [x] Instead of declaring variables on the global scope you should use the RSpec [`let`](https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let) method for declaring variables. If you declare your variable in global scope, it will get affected by the actions you make in your test cases. Therefore you will not be able to properly test your methods because of this. 

With the `let` method scope of your variable will be abstracted to each example (`it` block). 

```suggestion
  let(:inplay) {Game.new}
```

Reply:
Noted